install.py

```diff
@@ -95,14 +106,23 @@ def check_distribution_compatibility():
 
 
 def get_distribution_info():
-	# return distribution name and major version
-	if platform.system() == "Linux":
-		current_dist = platform.dist()
-		return current_dist[0].lower(), current_dist[1].rsplit('.')[0]
-
-	elif platform.system() == "Darwin":
-		current_dist = platform.mac_ver()
-		return "macos", current_dist[0].rsplit('.', 1)[0]
+
+    # return distribution name and major version
+
+    if platform.system() == 'Linux':
+        if int(platform.python_version_tuple()[0]
+               + platform.python_version_tuple()[1]) <= 3.7:
```

Comment:
You can use the `sys` module here

```suggestion
        if sys.version_info < (3, 7):
```